contrib/scripts/codepage_to_utf8.py

```diff
@@ -0,0 +1,38 @@
+#!/bin/python3
+"""
+Convert file from legacy codepage to Unicode UTF-8
+"""
+import argparse
+import os
+import sys
+
+parser = argparse.ArgumentParser(description="Convert file from legacy codepage to Unicode UTF-8",
+                                 formatter_class=argparse.RawTextHelpFormatter)
+parser.add_argument("filename", help="filename to convert")
```

Comment:
The `filename` argument's type can be upgraded from mere string to a read-only file, added after the help key, like: `help="...", type=argparse.FileType('r'))`.
                      
Also suggest adding the `required=True` key, because this is a necessary arg.

Reply:
The parser will take care of checking that the file exists and is readable, and you'll get the file object from the parser, so this eliminates some code further down.

Reply:
I will have a look at it the various comments over the weekend. Regarding the `required=True` key, that is only needed for non-positional arguments. positional arguments, which `filename` is, are automatically required.

Reply:
I had thought of it originally, but did not go down the path of using `argparse.FileType()`. I gave it a quick try and ran into various hurdles.

One problem is that the output file argument is currently optional, so while I can use it for the output file, I still need to handle the case where I need to open the input file as write. This just adds more code with no benefit.

Another problem is the input file, because at that point, I don't yet know the encoding. Unless I want to hard code the script to only handle cp437, which I don't want to do.